### DISCUSSION TITLE
[A-L] (2023/24) Foglio 10 - Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#213] [Esercizi - Bertini]
![immagine](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/bd6c42aa-2bce-48c1-9d64-c50c2450e02d)
<details>
<summary><b>Tavola integrale gaussiano</summary>
<img src=https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/9796be01-b51e-4448-a840-0f495ffeafab>
</details>

**Comment by Elia-Belli** [comment]
$X\sim \mathcal{W}(5,0.25)\longrightarrow Z=\frac{X-5}{0.5}\sim \mathcal{W}(0,1)$  con $X=\frac{1}{2}Z+5$

**(1)** La percentuale dei prodotti non accettabili è la probabilità che un prodotto sia di dimensioni inferiori a $4$ mm o superiori a $6$ mm:
$$\mathbb{P}(X\lt 4)+\mathbb{P}(X\gt 6)=\mathbb{P}(Z\lt -2)+\mathbb{P}(Z\gt 2)=^{(simmetria)}2\cdot \mathbb{P}(Z\gt 2)=$$

$$=2\cdot [1-\mathbb{P}(Z\leq 2)]=2\cdot [1-0.9772]=0.0456=4.56\ \\%$$

**(2)** Voglio trovare $\sigma^2$ affinchè la percentuale di pezzi non accettabili sia inferiore al $1\ \\%$, quindi pongo $Z=\frac{X-5}{\sigma}$ con $X=\sigma Z+5$ e svolgo di nuovo i calcoli imponendo alla fine la condizione per la probabilità desiderata:
$$\mathbb{P}(X\lt 4)+\mathbb{P}(X\gt 6)=2\cdot \mathbb{P}(Z\gt -\frac{1}{\sigma})=2\cdot [1-\mathbb{P}(Z\leq -\frac{1}{\sigma})]$$

$$2\cdot [1-\mathbb{P}(Z\leq -\frac{1}{\sigma})]\lt 0.01\to \mathbb{P}(Z\leq -\frac{1}{\sigma})>0.995$$

Quindi cerco nella tabella il valori più piccolo affinchè sia ottenga probabilità $>0.995$, ovvero $2.58$ e risolvo la seguente equazione in funzione di $\sigma$:

$$-\frac{1}{\sigma}=2.58\to \sigma = -\frac{1}{2.58}\Rightarrow \sigma^2=\frac{1}{6.6564}\simeq 0.15$$

**Comment by Mirko30** [comment]
Non mi è chiaro come nel 2) venga 2 * P(Z>-1/σ), non dovrebbe essere 2*P(Z>1/σ)?
